analysis_options.yaml

```diff
@@ -91,7 +90,6 @@ linter:
     - prefer_collection_literals
     - prefer_conditional_assignment
     - prefer_contains
-    - prefer_equal_for_default_values
```

Comment:
https://dart.dev/tools/linter-rules/prefer_equal_for_default_values

This has also been removed

Reply:
@diarmaidlindsay Exactly, that is why I am deleting it.

Reply:
Yep again I was supporting the removal by linking to the docs 👍